clang/lib/CodeGen/CGVTables.cpp

```diff
@@ -639,8 +639,16 @@ void CodeGenVTables::addRelativeComponent(ConstantArrayBuilder &builder,
   // want the stub/proxy to be emitted for properly calculating the offset.
   // Examples where there would be no symbol emitted are available_externally
   // and private linkages.
-  auto stubLinkage = vtableHasLocalLinkage ? llvm::GlobalValue::InternalLinkage
-                                           : llvm::GlobalValue::ExternalLinkage;
+  //
+  // `internal` linkage results in LOCAL Elf linkage while still manifesting a
```

Comment:
This is just a nit, but ELF is usually spelled in all caps since it's an acronym. I'd also use `STB_LOCAL` rather than `LOCAL` to be more explicit and use "binding" rather than "linkage".